ui/ui-expenses/src/screens/create/create.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Price from './price'
+import styles from './create.styles'
+
+const Create = ({
+  prices,
+  showInput,
+  toggleInput,
+  onChange,
+}) => (
+  <div className={styles.container}>
+    <h2>Prix</h2>
+    {!!prices.length && !showInput && (
+      <div className={styles.prices}>
+        <div className="list">
+          {prices.map(price => <Price key={price} price={price} />)}
+        </div>
+        <button onClick={toggleInput}>+</button>
+      </div>
+    )}
+
+    {(showInput || !prices.length) && (
```

Comment:
You can set showInput in store with the reaction binded to the HTTP request result.
`if (!prices || prices.length === 0) store.ui.create.showInput.set(true)`